src/effects.js

```diff
@@ -333,7 +333,10 @@ function Animation( elem, properties, options ) {
 		animation = deferred.promise({
 			elem: elem,
 			props: jQuery.extend( {}, properties ),
-			opts: jQuery.extend( true, { specialEasing: {} }, options ),
+			opts: jQuery.extend( true, {
+				specialEasing: {},
+				easing: jQuery.easing._default
+			}, options ),
```

Comment:
I think I'd prefer a default easing to stay at the Tween level, rather than here in Animate. And since @gnarf is back in this headspace for a bit, we can get a second opinion.